smoketest.py

```diff
@@ -76,7 +76,8 @@ def api_call(endpoint, verb, expected_status, payload=None):
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(
-        description='Performs API & Git smoketests against GitHub Enterprise Server')
+    description='Performs API & Git smoketests against GitHub Enterprise Server')
```

Comment:
I've been using `autopep8` for formatting - suspect that may be causing the difference here. Do you have a recommendation on a different formatter to use?

Reply:
I never use auto formatters and just tab it manually 🤷‍♀️